## ac-init.diff by Neil Williams <codehelp@debian.org>
##
## Bump AC_INIT to put the correct version into the .pc file

diff -urNad tslib-1.0~/configure.ac tslib-1.0/configure.ac
--- tslib-1.0~/configure.ac	2006-08-24 22:02:55.000000000 +0100
+++ tslib-1.0/configure.ac	2008-12-09 11:30:56.000000000 +0000
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
-AC_INIT(tslib, 0.0.2, kergoth@handhelds.org)
+AC_INIT(tslib, 0.1.0, kergoth@handhelds.org)
 # AC_CONFIG_AUX_DIR(config)
 AM_INIT_AUTOMAKE(dist-bzip2)
 AC_CONFIG_SRCDIR([src/ts_close.c])
